client/src/layouts/header/searchForm/cities/Cities.jsx

```diff
@@ -1,7 +1,15 @@
 import React from 'react';
+import classes from './cities.module.css';
+import City from './city/City';
 
-function Cities() {
-  return <div></div>;
+function Cities({ cities }) {
+  return (
+    <div className={classes.container}>
+      {cities.map((city, index) => (
+        <City key={`${city}${index}`} city={city} />
```

Comment:
Don't use index in key 
read more https://robinpokorny.medium.com/index-as-a-key-is-an-anti-pattern-e0349aece318

Reply:
i know, coz when we change our array it can cause some problems. but when i used city name, it says that i have not unique keys, maybe i have two the same cities
I will fix